### DISCUSSION TITLE
Exclude an enum type

### DISCUSSION THREAD

**Discussion by JohannesKaufmann** [ariga/atlas#2352] [Q&A] [answered]
I am trying to exclude the tables & types that [river](https://github.com/riverqueue/river) has generated. 

In my `atlas.hcl` in managed to **exclude** the tables (e.g. the "river_job" table). 

```hcl
env "local" {
    src = "file://db/schema.hcl"
    url = ...

    // Since the url contains "search_path=public" we dont need to prefix the glob with "public."
    exclude = ["river_*"]
}
```

But when I run inspect or apply, the enum type "river_job_state" is still included. 
```
-- Create enum type "river_job_state"
CREATE TYPE "river_job_state" AS ENUM ('available', 'cancelled', 'completed', 'discarded', 'retryable', 'running', 'scheduled');
```

-> What _glob pattern_ do I have to use to exclude an enum type?

**Comment by a8m** [comment, marked as answer]
Thanks for reporting this. https://github.com/ariga/atlas/pull/2356 added support for this and a new binary was pushed.

**Reply by JohannesKaufmann** [reply]
Wow, thanks for the quick fix! Now it works perfectly 🎉
